content/en/os/1.22.x/concepts/bootstrap-commands/_index.markdown

```diff
@@ -0,0 +1,128 @@
++++
+title = "Bootstrap Commands"
+type = "docs"
+description = "Setting up the host with Bottlerocket API commands during boot."
++++
+
+Bootstrap commands are a way to configure and modify a Bottlerocket instance using the Bottlerocket API at boot time. They run prior to the Bootstrap Containers.
```

Comment:
Should you add a link to the Bootstrap Containers documentation?

Reply:
Good callout. Added link.